docs/src/pages/de/installation.md

```diff
@@ -99,19 +99,20 @@ Du kannst jetzt den Platzhalter-Abschnitt "scripts" deiner `package.json` Datei,
 }
 ```
 
-Der Befehl [`dev`](#start-astro) startet den Astro Entwicklungs Server auf `http://localhost:3000`. Sobald dein Projekt fertig ist, generiert der Befehl [`build`](#build-astro) dein fertiges Projekt in das Verzeichnis `dist/`. [Lese mehr über das Veröffentlichen im Deployment Leitfaden.](/guides/deploy)
+Der Befehl [`dev`](#start-astro) startet den Astro Entwicklungsserver auf `http://localhost:3000`. Sobald dein Projekt fertig ist, gibt der Befehl [`build`](#build-astro) dein Projekt in das Verzeichnis `dist/` aus. [Lese mehr über das Veröffentlichen in der Anleitung zum Ausrollen](/guides/deploy).
```

Comment:
```suggestion
Der Befehl [`dev`](#start-astro) startet den Astro Entwicklungsserver auf `http://localhost:3000`. Sobald dein Projekt fertig ist, gibt der Befehl [`build`](#build-astro) dein Projekt in das Verzeichnis `dist/` aus. [Lese mehr über Veröffentlichen von Astro](/guides/deploy).
```
same game here 😊

Reply:
That is a hard one. It sounds so wrong, because I don't publish Astro, I am using Astro to compile something, and that is what I need to publish then by any other means.

So how about doing like so? [Lese mehr über das Veröffentlichen von Astro-Builds](/guides/deploy).

Reply:
Yea you are right. I personally don't mind having some English words in German sentences to describe stuff like building, deploying, compiling since .. I mean it's just the terminology and I think all the devs are used to that anyways.

Reply:
Consistency is king. So I would tend to write down a nomenclature to avoid confusion. Deploy seems to be unnecessary far fetched since we already have 'veröffentlichen' which is just exact enough. On the other hand there is nothing in German that compares well to build. So I would go with build. I will file these decisions some place to not loose track.